core/db/migrate/20210702112334_add_missing_timestamp_columns.rb

```diff
@@ -0,0 +1,37 @@
+class AddMissingTimestampColumns < ActiveRecord::Migration[6.1]
```

Comment:
@tomurb ,

We will need to use `[5.2]` here for the Rails 5.2 compatibility.
`class AddMissingTimestampColumns < ActiveRecord::Migration[5.2]`

Reply:
@MatthewKennedy fixed